docker/Dockerfile

```diff
@@ -31,48 +30,28 @@ RUN yum -y install curl wget bzip2 make git patch unzip bison yasm autoconf auto
 ENV PATH /opt/rh/devtoolset-2/root/usr/bin:$PATH
 ENV LD_LIBRARY_PATH /opt/rh/devtoolset-2/root/usr/lib64:/opt/rh/devtoolset-2/root/usr/lib
 
+# Shell scripts for build
+RUN git clone https://github.com/matthew-brett/manylinux.git -b refactor-dockerfile
```

Comment:
Docker is rather weird. Apparently the way it works is:
* Everything inside the directory that contains the Dockerfile is (recursively) uploaded to the docker daemon
* Any of those files which get mentioned in an `ADD` / `COPY` command get placed into the container
* All the rest of those files get thrown away

Therefore the rule is: put the Dockerfile in its own directory with just the stuff it needs, and then use `COPY`. (And apparently [`COPY` is preferred over `ADD`](https://docs.docker.com/engine/articles/dockerfile_best-practices/#add-or-copy).)